docs/specs/src/hwp_1_intro.md

```diff
@@ -0,0 +1,212 @@
+confidence indicates **social coherence**. This notion of social
+coherence is the single most important design goal of Holochain: to
+create the tooling that in contextually appropriate ways leads to
+increasing social coherence.
+
+Given the above, we propose a very simple approach to creating tooling
+capacity for building increasing certainty: **enable validated global
+visibility, on demand, of local state**. In this approach, we
+distinguish between authorship, which is about local state changes
+initiated by agents, and authority, which is about distributing the
+responsibility of validating and making visible those state changes
+across the participants in the network. This approach requires that we:
+
+1.  Ensure that all agents know the "ground-rules". i.e. offer a
+  framework for composing lots of small units of social agreement in
+  which players can add deterministic elements of certainty into
+  their interactions.
+
+2.  Ensure that all agents can *reliably* see what's going on. i.e.
+  offer a framework for adding to and querying a collectively held
+  database in which there is a knowable level of confidence in its
+  reliability and integrity that can also be graduated to fit the
+  circumstances.
+
+The first point we deliver through various types of **Validation
+Rules**. Validation rules create certainty in the following dimensions
+with examples:
+
+-   Content: a value does not exceed a maximum length
+
+-   Structure: an entry consists of a certain set of types of data
+
+-   Sequence: someone can not spend credits they have not already
+  received earlier on their chain
+
+-   Process: a transaction must be approved and signed by a notary
+
+-   Behavior: one does not take an action more frequently than a certain
+  rate
+
+The second point we deliver through various types of **Intrinsic Data
+Integrity**. We use a number of cryptographic methods to create
+self-proving data of various types:
+
+-   Provenance: An agent's network address is their public-key. Thus,
+  when interacting with agents it's possible to have deterministic
+  confidence in who one is interacting with because there is no
+  identity layer between network locations subject to attack
+  surface. I.e. Unlike the web address, you don't need a domain name
+  certificate associated with the domain name to become confident of
+  "who" you are talking to.
+
+-   Signatures: Because provenance is a public key, it's also easy to
+  create self-proving authenticity. All messages sent, and all data
+  committed to chains is signed by agents using their public key.
+  Thus any agent can immediately and with high confidence, verify
+  the authenticity of messages and data.
+
+-   Hashes: All data on our DHT is addressed by its hash. Thus, when
+  retrieving data it's possible to have deterministic confidence
+  that it hasn't been tampered with by whoever was storing it.
+
+-   Structural Monotonicity (for chains and DHT): non-repudiable
```

Comment:
repudiable exists in Spanish, what is this in English? non-discardable?

Reply:
un-deniable